app/controllers/works_controller.rb

```diff
@@ -117,7 +117,7 @@ def complete_globus_setup
 
     authorize! work_version, to: :show?
 
-    if GlobusClient.user_exists?(work.depositor.sunetid)
+    if globus_user_exists?(work.depositor.email)
       GlobusSetupJob.perform_later(work_version)
```

Comment:
note: i noticed this is actually a small bug in the recent implementation ... we actually need to send the email address to the globus client and not just the sunet

Reply:
Good catch. That was one of the breaking changes between 0.4.0 and 0.5.0.